README.md

```diff
@@ -28,31 +28,25 @@ refresh = true
 misbehaviour = true
 
 [mode.connections]
-enabled = true
+enabled = false
 
 [mode.channels]
-enabled = true
+enabled = false
 
 [mode.packets]
 enabled = true
 clear_interval = 100
 clear_on_start = true
 tx_confirmation = true
 
-[telemetry]
-enabled = true
-host = '127.0.0.1'
-port = 3001
-
 [[chains]]
-id = 'ibc-eth-0'
+id = 'ibc-eth-1'
 genesis_time = 1606824023
 genesis_root = "0x4b363db94e286120d76eb905340fdd4e54bfe9f06bf33ff6cf5ad27f511bfe95"
-initial_checkpoint = "0x428ce0b5f5bbed1fc2b3feb5d4152ae0fe98a80b1bfa8de36681868e81e9222a"
-key_name = 'wallet'
+initial_checkpoint = "0x6c668d888d2b892bf26c25ee937ed43b48048dedda971ff33ebaaae7b2bd3890"
+key_name = 'relayer_eth_wallet'
 rpc_addr_pool = [
-  "https://www.lightclientdata.org",
-  "https://beacon-nd-995-871-887.p2pify.com/c9dce41bab3e120f541e4ffb748efa60"
+  "http://127.0.0.1:8052",
```

Comment:
Is this local RPC address intended?

Reply:
no matter what it is, but restoring it may be a better option